### ISSUE TITLE
Finish code on name resolving and cleaning phylo data

### ISSUE THREAD

**Issue by diazrenata** [role-model/multidim-biodiv-data#126]

**Comment by diazrenata** [comment]
Phylo episode thinking....

* pulling the entire tree of arthropods is a Lot, but fun for visualization... the thing is that I get a "request too large" error for all arthropods/arachnids. I was able to get a tree for orb weavers to download and plot, but now I question the pedagogical value of that anyway. 
* ultimately here we're optimizing for the flow of the lesson itself.... --> perhaps we pull a tree for orb weavers, then do the tip matching/tip adding exercise for the orb weavers in the hawaii data? Or, _don't_ bother with pulling a tree for all taxonomic groups, and just using the rotl name resolver to navigate naming issues. 

ok well name_resolve on all orb weavers is silly slow

I just tried

* rotl match_names --> getting a phylo --> taxize gnr_resolve on the phylo tip names

this produced no changes in the actual names, once you clean off the ottIDs and taxonomic notes. 

Is it needed, to have a name resolve exercise here? perhaps not.
